src/scss/test-results-list.component.scss

```diff
@@ -0,0 +1,56 @@
+div#test-results-list {
+  th {
+    &.col-id {
+      width: 6rem;
+    }
+
+    &.col-name,
+    &.col-status {
+      width: 15rem;
+    }
+  }
+}
+
+table.detail-list {
+  font-size: 0.88rem;
+  font-weight: 0.85rem;
+  padding: 1rem;
+
+  th {
+    padding: 0.7rem 1.2rem;
+  }
+
+  tr {
+    border-bottom: 0.05rem solid gray;
+
+    &[role="button"]:hover {
+      background-color: #292d30;
+      cursor: pointer;
+    }
+  }
```

Comment:
Lots of styling applied to HTML elements and on attribute filters. Would rather see it applied solely by classes, as that usually results in better reusability, cleaner stylesheets, and less glitch- and regression-prone.